deploy/gatekeeper.yaml

```diff
@@ -2488,6 +2488,10 @@ spec:
         kubernetes.io/os: linux
```

Comment:
Same as `charts/`, we should undo changes in `deploy/`

Reply:
reverted